src/main/kotlin/no/nav/syfo/behandling/InntektsmeldingBehandler.kt

```diff
@@ -36,6 +36,7 @@ class InntektsmeldingBehandler(
         val log = log()
         log.info("Henter inntektsmelding for $arkivreferanse")
         val inntektsmelding = journalpostService.hentInntektsmelding(arkivId, arkivreferanse)
+        inntektsmeldingService.isDuplicate(inntektsmelding)
```

Comment:
Gjør den noe?

Reply:
Foreløpig skal vi logge for å finne ut hvor ofte det skjer. Er det hyppig så skal vi ignorere de.